src/views/NotFound.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <center-wrapper>
+    <h2>Invalid URL</h2>
+    <p>Uh-oh, this URL does not exist.</p>
+    <router-link to="/">Return home</router-link>
+  </center-wrapper>
+</template>
+
+<script>
+import CenterWrapper from '@/components/CenterWrapper.vue'
+
+export default { components: { CenterWrapper } }
```

Comment:
nit: considering create a prop `title` so that different routes can display different messages